translator/totomlconfig/toTomlConfig.go

```diff
@@ -58,9 +58,9 @@ func ToTomlConfig(c interface{}) string {
 	_, val := r.ApplyRule(c)
 	buf := bytes.Buffer{}
 	enc := toml.NewEncoder(&buf)
-	e := enc.Encode(val)
-	if e != nil {
-		panic(e)
+	err := enc.Encode(val)
+	if err != nil {
+		panic(err)
```

Comment:
Why don't we log this err?

Reply:
Cause I thought [this log](https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/cmdutil/translatorutil.go#L45-L47) would be enough for us to track down  the issue. However, it could be from `BurntSushi` itself so reconsider to add the error.